QuestionsforKaida_s2.md

```diff
@@ -0,0 +1,6 @@
+- How can I refactor the code without repeating the  footer and nav for each page?
+- What is the best way to import google fonts? link or @url import 
+- How to target the right element? When should I target the parent and when should I target the child? sometimes it seems interchangeable
+- Which elements should be "stored" in containers?
+- Why sometimes  my font-family Garland is applied and sometimes not?
```

Comment:
I'm not totally sure.  My guess is that it's because you load it after you load your global "style.css" and sometimes that loads before the fonts finish loading.  

(If you use an import or download the font and check it into Git then you won't have to worry about things like download speeds affecting your layout.)